app/src/main/java/org/mozilla/reference/browser/browser/FindInPageIntegration.kt

```diff
@@ -45,7 +46,7 @@ class FindInPageIntegration(
     }
 
     private fun launch() {
-        val session = sessionManager.selectedSession ?: return
+        val session = sessionId?.let { sessionManager.findSessionById(it) } ?: sessionManager.selectedSession ?: return
```

Comment:
nit: Maybe `runWithSessionIdOrSelected` could be a bit clear

Reply:
Good point, fixed!